diff/deleter.go

```diff
@@ -130,11 +129,6 @@ func (d *Deleter) deleteRelation(id int64, deleteRefs bool, deleteMembers bool)
 		if m.Way == nil {
 			continue
 		}
-		err := d.osmCache.Coords.FillWay(m.Way)
```

Comment:
It loads the coordinates from the cache for all nodes of this way. It's possible that they are not set at this stage.